.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I love the idea of the recommendations, but I am not 100% sure about this file. The `settings.json` *mandates* this setting to be on, but people may not like these things to be forced on them.

I know some projects use Git hooks to ensure that styling is consistent. Maybe we can look at that? Although I think it's not really worth it.